Do work in the wdl working directory

ttung opened the pull request:
### Description
This facilitates copying of data out of the runs if need be.

Depends on #501 

### Test plan
started a run.  It's not yet done.

danrlu commented on .happy/terraform/modules/sfn_config/nextstrain.wdl:
```diff
@@ -68,36 +68,37 @@ task nextstrain_workflow {
+    (cd ncov &&
+     git init &&
+     git fetch --depth 1 git://github.com/nextstrain/ncov.git &&
+     git checkout FETCH_HEAD
```
i've been wanting to ask: what does this do? to get the hash?

ttung replied:
which line?

danrlu replied:
`git checkout FETCH_HEAD`

ttung replied:
When one runs `git fetch`, it creates a temporary branch called "FETCH_HEAD" that represents the HEAD of what you just fetched.

danrlu commented on .happy/terraform/modules/sfn_config/nextstrain.wdl:
```diff
@@ -68,36 +68,37 @@ task nextstrain_workflow {
+    aws s3 cp --no-progress "s3://${aligned_gisaid_s3_bucket}/${aligned_gisaid_metadata_s3_key}" ncov/data/metadata_gisaid.tsv
 
-    snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/  --resources=mem_mb=312320 || aws s3 cp /ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_id}/" --recursive
+    (cd ncov && snakemake --printshellcmds auspice/ncov_aspen.json --profile my_profiles/aspen/ --resources=mem_mb=312320) || aws s3 cp ncov/.snakemake/log/ "s3://${aspen_s3_db_bucket}/phylo_run/${build_id}/" --recursive
```
would `cd ncov` here influence the following `aws s3 cp ncov/...` and in line 101, 119 or not?

ttung replied:
No, it's in a subshell, so it doesn't affect the rest.

danrlu reviewed (approved):
I tested out this version of nextstrain-dbg.wdl too. Works beautifully!

Commit:
Do work in the wdl working directory

### Description
This facilitates copying of data out of the runs if need be.

### Test plan
started a run.  It's not yet done.

danrlu commented:
Checks are not moving. I'll open a new PR.
